docs/050-breaking-changes.rst

```diff
@@ -369,8 +370,8 @@ Old version:
 
 ::
 
-    // This will not compile
     pragma solidity ^0.4.25;
+    // This will not compile
```

Comment:
It does compile for 0.4.25, does it? Can we do it like in the example above?

Reply:
It doesn't compile with 0.4.25, but I can check why and fix it

Reply:
Solved now